### DISCUSSION TITLE
Version .NET used in CSharpRepl

### DISCUSSION THREAD

**Discussion by Wavesolid** [waf/CSharpRepl#263] [Q&A]
Is that possible to change version .NET in this REPL? Let say in current version is .NET 7 and I want to switch to .NET 8 preview or even change to .NET 6.

**Comment by awa5114** [comment]
Would be great to get an answer on this. I'm loading a third-party DLL which was built with .NET 6 and get compatibility issues because `csharprepl` apparently only runs with .NET 7 on my machine. Would be nice to know if this can be made to work.

**Reply by awa5114** [reply]
@waf would it be possible to please get a response on this? I'm finding this REPL extremely useful and it would be awesome to also be able to load in DLLs for .NET <7

**Reply by waf** [reply]
@amine-aboufirass I'm definitely open to supporting this, though I'm not entirely sure how much effort it would be.

What are the compatibility issues you're getting? Does it help if you pass the command line flag `--framework Microsoft.NETCore.App/6.0.10`, where that 6.0.10 version is whatever version is available in your .NET installation directory (on Window, that's e.g. `C:\Program Files\dotnet\shared\Microsoft.NETCore.App`)?
